chibuezeayogu/Events-Manager
# #160008810 Implement user login

**What does the PR do?**
- This PR implements user `Login`

**Description of Task to be completed?**
- Ensure users can log in
- Ensure that token is generated when users logs in 
- Ensure test is passing 

**How should this be manually tested?**
- Clone the repository [Events-Manager](https://github.com/chibuezeayogu/Events-Manager.git) and
- Checkout to  this branch `ft-user-signin-#160008810`
- run `bundle install` to install new `rails gems`
- run `rails server` and test this route ` Post /api/vi/users/login` using `Postman`
- run `rspec` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160008810](https://www.pivotaltracker.com/story/show/160008810)

**Screenshots**
<img width="1356" alt="screen shot 2018-08-27 at 1 25 23 am" src="https://user-images.githubusercontent.com/26305570/44634988-34c13300-a998-11e8-85eb-08fbcd1302e2.png">
<img width="1358" alt="screen shot 2018-08-27 at 1 26 02 am" src="https://user-images.githubusercontent.com/26305570/44634991-3be84100-a998-11e8-834d-7bf5a7ed11dd.png">
